src/DataSift/TestRest/BaseContext.php

```diff
@@ -155,6 +155,17 @@ public static function setupEnvironment()
     }
 
     /**
+     * Setup the database after every scenario (if configured).
+     * The database settings are defined in behat.yml
+     *
+     * @AfterScenario
```

Comment:
I think a better option would be to remove the `@BeforeScenario` in place of the `@BeforeFeature`.